packages/compass-e2e-tests/tests/oidc.test.ts

```diff
@@ -83,7 +73,6 @@ describe('OIDC integration', function () {
       }
     }
 
-    // TODO(MONGOSH-1306): Get rid of all the setup code to download mongod here... :(
```

Comment:
We probably still don't want to set up a separate server that doesn't match the one we are supposedly running tests against inside the test setup here. Probably worth creating a different ticket for that?

Reply:
I get that it’s not ideal, but I’m honestly not quite sure what a better way is. I’ve thought about using the same model that we use in mongosh (and other Compass packages in this PR), i.e. spawning servers at the sites where they are being used, and discussed it with @lerouxb but that’s quite a departure from how compass-e2e-tests currently works (expects setup to be separate from mocha hooks, expects ability to run against existing server, etc.)

So I’m going a bit back and forth here myself…

Reply:
I guess we could, but it would make the code more complex and less local … so I think it would be good to know what the benefits are (starting and stopping servers is ultimately actually fairly fast, it’s only download and extraction that may take a while)